app/helpers/otp_helper.rb

```diff
@@ -0,0 +1,44 @@
+module OtpHelper
+  def phone_confirmation_instructions
+    if @delivery_method == 'sms'
+      t('instructions.2fa.confirm_code_sms', number: @phone_number)
+    elsif @delivery_method == 'voice'
+      t('instructions.2fa.confirm_code_voice', number: @phone_number)
+    end
+  end
+
+  def alternative_2fa_links
+    case @delivery_method
+    when 'sms'
+      "#{voice_fallback_link}#{totp_option_link}."
```

Comment:
I guess not, if we know the strings are 100% not user-supplied? However if we send one of the i18n strings with `<strong>` in it off to get translated and it comes back missing a `>` or something, we've just broken our app. I've had this happen to me before and so I prefer to remain extra vigilant about that (hence my motivation to remove HTML markup from i18n strings entirely).

I'd rather not have to treat each string differently when interpolating/combining, and just be extremely careful with each one of them? I guess the shortest way to explain how I feel, `.html_safe` is a code smell to me, and `==` in our templates is basically a shorthand for `.html_safe` which is why I am wary of it.

If it's too much hassle to work around, or I get outvoted, I totally understsand